cluster-autoscaler/cloudprovider/gce/gce_cloud_provider.go

```diff
@@ -172,7 +172,11 @@ func (gce *GceCloudProvider) NewNodeGroup(machineType string, labels map[string]
 		},
 		gceManager: gce.gceManager,
 	}
-	_, err := gce.gceManager.getTemplates().buildNodeFromAutoprovisioningSpec(mig)
+	cpu, mem, err := mig.gceManager.getCpuAndMemoryForMachineType(mig.spec.machineType, mig.GceRef.Zone)
```

Comment:
Why are we now refering to gceManager via mig? I guess it doesn't really matter, but it's a strange thing to change.

Reply:
Fixed. It was copy and paste.

Reply:
done